presidio-analyzer/tests/test_spacy_recognizer.py

```diff
@@ -1,13 +1,17 @@
+import pytest
 from unittest import TestCase
 
-from tests import assert_result, assert_result_within_score_range, TESTS_NLP_ENGINE
+from tests import assert_result, assert_result_within_score_range
 
-from presidio_analyzer.predefined_recognizers import SpacyRecognizer
+from presidio_analyzer.nlp_engine import NLP_ENGINES
+from presidio_analyzer.predefined_recognizers import NLP_RECOGNIZERS
 from presidio_analyzer.entity_recognizer import EntityRecognizer
 
 NER_STRENGTH = 0.85
-nlp_engine = TESTS_NLP_ENGINE
-spacy_recognizer = SpacyRecognizer()
+nlp_is_available = "spacy" in NLP_ENGINES and "spacy" in NLP_RECOGNIZERS
```

Comment:
should be `NLP_ENGINES` or `TESTS_NLP_ENGINE`?

Reply:
I refactored everything to use pytest fixtures.  You can see `tests/conftest.py` to see what I did.  There is some logic to load the small spacy model if `--runfast` is used.